txscript/standard.go

```diff
@@ -1150,22 +1150,29 @@ func PayToAddrScript(addr dcrutil.Address) ([]byte, error) {
 }
 
 // MultiSigScript returns a valid script for a multisignature redemption where
-// nrequired of the keys in pubkeys are required to have signed the transaction
-// for success.  An Error with kind ErrTooManyRequiredSigs will be returned if
-// nrequired is larger than the number of keys provided.
-func MultiSigScript(pubkeys []*dcrutil.AddressSecpPubKey, nrequired int) ([]byte, error) {
-	if len(pubkeys) < nrequired {
+// the specified threshold number of the keys in the given public keys are
+// required to have signed the transaction for success.  An Error with kind
+// ErrTooManyRequiredSigs will be returned if the number required is larger than
+// the number of keys provided.
```

Comment:
I should call out that the public keys must be serialized in the compressed format here too.

Reply:
Done.